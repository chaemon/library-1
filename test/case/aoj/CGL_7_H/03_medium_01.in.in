58 70
10 0
50 0
50 10
80 10
80 20
70 20
70 60
60 60
60 20
50 20
50 70
40 70
40 50
20 50
20 90
10 90
10 40
40 40
40 10
20 10
20 30
-10 30
-10 60
-20 60
-20 30
-90 30
-90 20
-60 20
-60 -80
-50 -80
-50 20
-40 20
-40 -40
-10 -40
-10 -30
-30 -30
-30 20
-10 20
-10 10
-20 10
-20 0
-10 0
-10 -20
10 -20
10 -50
-30 -50
-30 -60
20 -60
20 -80
30 -80
30 -50
20 -50
20 -30
70 -30
70 -20
20 -20
20 -10
10 -10
