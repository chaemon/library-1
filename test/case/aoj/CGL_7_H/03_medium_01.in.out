6654.1353369023
